doc/source/tokudb/tokudb_variables.rst

```diff
@@ -949,6 +959,32 @@ case there are bugs. Not for use in production.
 Defines direction to be used to perform table scan to check for empty tables
 for bulk loader.
 
+.. variable:: tokudb_enable_fast_update
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Global/Session
+  :dyn: Yes
+  :vartype: Boolean
+  :default: OFF
+
+Toggles the fast updates feature ON/OFF for the ``UPDATE`` statement. Fast
```

Comment:
The previous comments still stand: what's there in this description to prevent users from turning it always on?

Reply:
mentioned that fast updates feature is experimental